contrib/openshift/manifests/manifests.yaml

```diff
@@ -391,3 +400,16 @@ parameters:
     value: "clair"
     displayName: clair service account
     description: name of the service account to use for API interaction
+  # DVO
+  - name: CLAIR_DISABLE_MIN_REPLICAS_CHECK
+    displayName: Disable Minimum Replicas Check
+    value: "min-replicas-check-not-disabled"
+  - name: CLAIR_DISABLE_MIN_REPLICAS_REASON
+    displayName: Reason for Disabling Minimum Replicas Check
+    value: ""
+  - name: CLAIR_DISABLE_ANTI_AFFINITY_CHECK
+    value: "anti-affinity-check-not-disabled"
+    displayName: disable DVO check for anti-affinity
+  - name: CLAIR_DISABLE_ANTI_AFFINITY_CHECK
```

Comment:
Is this variable supposed to be repeated or should it be `CLAIR_DISABLE_ANTI_AFFINITY_REASON`?